deepchecks/nlp/utils/nlp_plot.py

```diff
@@ -230,3 +232,21 @@ def get_text_outliers_graph(dist: Sequence, data: Sequence[str], lower_limit: fl
         hoverdistance=-1)
 
     return fig
+
+
+def two_datasets_scatter_plot(plot_title, plot_data, test_dataset, train_dataset, dataset_names):
+    """Plot a scatter plot of two datasets."""
```

Comment:
What is plot data? Document for future selves

Reply:
Done